psydac/linalg/tests/test_stencil_matrix.py

```diff
@@ -2001,193 +2001,6 @@ def test_stencil_matrix_2d_serial_backend_dot_3(dtype, n1, n2, p1, p2, s1, s2, P
     assert np.allclose(y1a, y1a_exact, rtol=1e-13, atol=1e-13)
     assert np.allclose(y2a, y2a_exact, rtol=1e-13, atol=1e-13)
 
-# ===============================================================================
-@pytest.mark.parametrize('dtype', [float,complex])
-@pytest.mark.parametrize('n1', [5])
-@pytest.mark.parametrize('p1', [2])
-@pytest.mark.parametrize('s1', [1])
-@pytest.mark.parametrize('P1', [False])
-@pytest.mark.parametrize('backend', [PSYDAC_BACKEND_PYTHON, PSYDAC_BACKEND_NUMBA, PSYDAC_BACKEND_GPYCCEL])
-def test_stencil_matrix_1d_serial_backend_transpose(dtype, n1, p1, s1, P1, backend):
-    # Create domain decomposition
-    D = DomainDecomposition([n1 - 1], periods=[P1])
-
-    # Partition the points
-    npts1 = [n1]
-    global_starts1, global_ends1 = compute_global_starts_ends(D, npts1, [p1])
-
-    npts2 = [n1 - 1]
-    global_starts2, global_ends2 = compute_global_starts_ends(D, npts2, [p1])
-
-    cart1 = CartDecomposition(D, npts1, global_starts1, global_ends1, pads=[p1], shifts=[s1])
-    cart2 = CartDecomposition(D, npts2, global_starts2, global_ends2, pads=[p1], shifts=[s1])
-
-    # Create vector space, stencil matrix, and stencil vector
-    V1 = StencilVectorSpace(cart1, dtype=dtype)
-    V2 = StencilVectorSpace(cart2, dtype=dtype)
-    M1 = StencilMatrix(V1, V2, pads=(p1 - 1,), backend=backend)
-    M2 = StencilMatrix(V2, V1, pads=(p1 - 1,), backend=backend)
-
-    # Fill in stencil matrix values based on diagonal index (periodic!)
-    if dtype == complex:
-        M1[0:n1, :] = np.random.random((n1, 2 * p1 - 1)) \
-                                       +1j*np.random.random((n1, 2 * p1 - 1))
-        M2[0:n1 - 1, :] = np.random.random((n1 - 1, 2 * p1 - 1)) \
-                                   +1j*np.random.random((n1 - 1, 2 * p1 - 1))
-    else:
-        M1[0:n1, :] = np.random.random((n1, 2 * p1 - 1))
-        M2[0:n1 - 1, :] = np.random.random((n1 - 1, 2 * p1 - 1))
-    # If any dimension is not periodic, set corresponding periodic corners to zero
-    M1.remove_spurious_entries()
-    M2.remove_spurious_entries()
-
-    # Convert stencil objects to Numpy arrays
-    M1a = M1.toarray()
-    M2a = M2.toarray()
```

Comment:
Does the `tosparse` method use an accelerated kernel?

Reply:
Yes the `toarray` and the `tosparse` method both used the kernels `stencil2coo_{dim}d_{order}` EXCEPT in the case were we want to get the pads in parallel mode